packages/terra-base/src/Base.jsx

```diff
@@ -79,7 +79,7 @@ class Base extends React.Component {
     if (locale === undefined) return childComponent;
     if (!this.state.areTranslationsLoaded) return <div>{this.props.translationsLoadingPlaceholder}</div>;
     return (
-      <I18nProvider locale={this.state.locale} messages={messages}>
+      <I18nProvider locale={this.state.locale} messages={messages} {...customProps}>
         {childComponent}
```

Comment:
This wrapping div should be removed, and children should be passed into I18nProvider directly. The provider adds it's own wrapping div.
<div {...customProps}>
        {children}
</div>

Also i18nProvider doesn't use customProps, so that needs to be added to the provider component as well.

Reply:
Updated https://github.com/cerner/terra-core/commit/92f4f7cb44763c557144cc87069b86448238ec6b

Reply:
Updated https://github.com/cerner/terra-core/commit/7e2e1693e35aa1ee1404b224eb2671d7de9420ef

Reply:
Updated with customProps to be the first attribute https://github.com/cerner/terra-core/commit/1c7fc9f6a3953ac9212d8fd4483d8e035f2c7b03